packages/xrpl/src/sugar/submit.ts

```diff
@@ -52,12 +52,73 @@ async function submit(
  * validated ledger (or has errored/will not be included for some reason).
  * See [Reliable Transaction Submission](https://xrpl.org/reliable-transaction-submission.html).
  *
+ * @example
+ *
+ * ```ts
+ * const { Client, Wallet } = require('xrpl')
+ * const client = new Client('wss://s.altnet.rippletest.net:51233')
+ *
+ * async function submitTransaction() {
+ *   const senderWallet = Wallet.generate()
+ *   const recipientWallet = Wallet.generate()
```

Comment:
As is this code won't work because you're trying to send money from an account which is newly generated, and doesn't have any funds yet. 

While for the presentation I think it's valuable to walk through the `Wallet.generate()` differences, I don't think we should do that in the core submit flow. (As there people just want it to work)

```suggestion
 *   const senderWallet = client.fundWallet()
 *   const recipientWallet = client.fundWallet()
```